js/tc39/tc39_test.go

```diff
@@ -503,9 +508,10 @@ func (ctx *tc39TestCtx) runTC39File(name string, t testing.TB) {
 		// t.Logf("Running strict test: %s", name)
 		ctx.runTC39Test(t, name, src, meta, true)
 	} else { // Run test in non strict mode only if we won't run them in strict
+		// NOTE: in practice k6 does not run non strict code at all so this tests make no sense
 		// TODO uncomment the if above and delete this else so we run both parts when the tests
 		// don't take forever
-		ctx.runTC39Test(t, name, src, meta, false)
+		// ctx.runTC39Test(t, name, src, meta, false)
```

Comment:
hmm ok, but does running these result in a ton more failures? or does it just take longer? 

I don't have a firm opinion on whether we should run or should not run these, but if we aren't going to ever run them, my vote would be to delete the whole `else` clause and just comment about strict mode above

Reply:
reenabled it but dropped the generator ones with [e524b0e](https://github.com/grafana/k6/pull/2830/commits/e524b0e8e926bc0a7a6d181ed9ce513a84c561ed)